src/math/extgcd.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <bits/stdc++.h>
+using namespace std;
+
+template <class T>
+T extgcd(T a, T b, T& x, T& y) {
+    if (b > 0) {
```

Comment:
```suggestion
    if (b != 0) {
```
b < 0 を許容する場合、この変更が必要だと思います。

Reply:
なるほど... ありがとうございます！   [`bfdf3c4`](https://github.com/YNUCPC/ynu-icpc-library/pull/50/commits/bfdf3c45a5d3d6c71e872925047a43713ece34a2) で対応しました 🙇